proto/pairing/epoch_payments.proto

```diff
@@ -6,6 +6,6 @@ import "pairing/provider_payment_storage.proto";
 
 message EpochPayments {
   string index = 1; 
-  repeated ProviderPaymentStorage clientsPayments = 2; 
+  repeated string providerPaymentStorageKeys = 2; 
```

Comment:
why is the name changed from clients to providers?

Reply:
The "clientPayments" field was of type providerPaymentStorage. So Omer told me to change the name accordingly